### PR TITLE
ADD: Change base model to RoBERTa

### PR DESCRIPTION
**Pull request by benleetownsend**

### CODE REVIEW

File: finetune/config.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 import finetune
 from finetune.errors import FinetuneError
-from finetune.base_models import GPTModel, GPT2Model, BERT, TextCNN
```
Comment (benleetownsend) [inline finetune/config.py:14]: I don't see a reason why we'd need this??

Comment (madisonmay) [reply finetune/config.py:14]: Yeah this is unnecessary.

**Comment by madisonmay** [comment]
I'm guessing we're going to get some failing tests that are written assuming default base model is GPT.

**Comment by benleetownsend** [comment]
The following tests are failing.
```
<tests.test_sequence.TestSequenceLabelerNoCRF testMethod=test_chunk_long_sequences>
<tests.test_sequence.TestSequenceLabeler testMethod=test_chunk_long_sequences>
<tests.test_parity.TestActivationParity testMethod=test_gpt_multifield_parity>
<tests.test_ordinal.TestOrdinal testMethod=test_reasonable_predictions>
<tests.test_mtl.TestMTL testMethod=test_fit_predict>
<tests.test_classifier.TestClassifier testMethod=test_explain
```

**Pull request merged**